### PR TITLE
config: expose lock compact interval

### PR DESCRIPTION
**Pull request by BusyJay**
@siddontang @zhangjinpeng1987 PTAL

### CODE REVIEW

File: src/raftstore/store/config.rs [hunk 1]
```diff
@@ -28,8 +28,8 @@ const SPLIT_REGION_CHECK_TICK_INTERVAL: u64 = 10000;
 const REGION_SPLIT_SIZE: u64 = 64 * 1024 * 1024;
 const REGION_MAX_SIZE: u64 = 80 * 1024 * 1024;
 const REGION_CHECK_DIFF: u64 = 8 * 1024 * 1024;
-const REGION_COMPACT_CHECK_TICK_INTERVAL: u64 = 300_000;
-const REGION_COMPACT_DELETE_KEYS_COUNT: u64 = 1_000_000;
+const REGION_COMPACT_CHECK_TICK_INTERVAL: u64 = 300;
+const REGION_COMPACT_DELETE_KEYS_COUNT: u64 = 200_000;
```
Comment (siddontang) [inline src/raftstore/store/config.rs]: I think exporting these and use them directly in tikv-server is better to avoid default configurations mismatched.

Comment (BusyJay) [inline src/raftstore/store/config.rs]: Maybe in another pr.

**Comment by siddontang** [comment]
LGTM, but CI failed.

File: src/bin/tikv-server.rs [hunk 2]
```diff
@@ -381,16 +381,21 @@ fn build_cfg(matches: &Matches, config: &toml::Value, cluster_id: u64, addr: &st
...
         get_toml_int(config,
-                     "raftstore.region-compact-check-tick-interval",
-                     Some(300_000)) as u64;
+                     "raftstore.region-compact-check-interval-secs",
+                     Some(300)) as u64;
 
     cfg.raft_store.region_compact_delete_keys_count =
         get_toml_int(config,
                      "raftstore.region-compact-delete-keys-count",
                      Some(200_000)) as u64;
```
Comment (BusyJay) [inline src/bin/tikv-server.rs]: Any suggestions?

Comment (siddontang) [inline src/bin/tikv-server.rs]: origin is 1_000_000

Comment (zhangjinpeng1987) [inline src/bin/tikv-server.rs]: 1000_000

**Comment by zhangjinpeng1987** [comment]
LGTM

**Pull request merged**